frontend/package.json

```diff
@@ -17,13 +17,19 @@
     "precommit": "lint-staged"
   },
   "dependencies": {
+    "@ledgerhq/devices": "^7.0.0",
     "@metamask/jazzicon": "^2.0.0",
     "@quasar/extras": "^1.0.0",
     "@umbra/umbra-js": "^0.1.0",
     "@uniswap/token-lists": "^1.0.0-beta.19",
     "@vue/composition-api": "^0.6.4",
+    "@web3-onboard/coinbase": "^2.1.1",
```

Comment:
_(arbitrary comment location to start a thread)_

Just noting that another wallet-related issue we have is #374, which we can do after this PR